src/Http/Client/Adapter/Stream.php

```diff
@@ -293,7 +293,7 @@ protected function _send(Request $request)
      */
     protected function _open($url)
     {
-        set_error_handler([$this, '_connectionErrorHandler']);
+        set_error_handler([$this, 'connectionErrorHandler']);
```

Comment:
I wonder if this should be a closure instead... So that we don't leave a public method around for no good reason